mobile/lib/services/video_preview_service.dart

```diff
@@ -70,21 +70,13 @@ class VideoPreviewService {
 
   late final SharedPreferences _prefs;
   static const String _videoStreamingEnabled = "videoStreamingEnabled";
-  static const String _videoStreamingCutoff = "videoStreamingCutoff";
 
   bool get isVideoStreamingEnabled {
-    return _prefs.getBool(_videoStreamingEnabled) ?? false;
+    return _prefs.getBool(_videoStreamingEnabled) ?? true;
```

Comment:
Let’s keep it off till we are sure it’s not crushing the app performance or crashing the app cc @setalp